feature/security/gnsi/credentialz/tests/password_console_login/README.md

```diff
@@ -0,0 +1,38 @@
+# Credentialz-1: Password console login
+
+## Summary
+
+Test that Credentialz properly creates users and the associated password and that the DUT handles
+authentication of those users properly.
+
+
+## Procedure
+
+* Set a username of `testuser` with a password of `i$V5^6IhD*tZ#eg1G@v3xdVZrQwj` using gnsi.Credentialz
```

Comment:
Instead of defining a static passwd here, why not define some restrictions:
  * must be 24-32 characters long
  *  must use 4 of the 5 character classes ([a-z], [A-Z], [0-9], [!@#$%^&\*(){}\[\]\\|:;'"], [ <tab>])
 
and provide a function that generates a passwd to be used (and verified) in` the test  ?

Reply:
Updated readme & added helper function to generate password